contrib/opencensus-ext-azure/tests/test_azure_log_exporter.py

```diff
@@ -171,6 +176,8 @@ def test_exception_with_custom_properties(self, requests_mock):
         self.assertTrue('ZeroDivisionError' in post_body)
         self.assertTrue('key_1' in post_body)
         self.assertTrue('key_2' in post_body)
+        self.assertTrue('measure_1' in post_body)
```

Comment:
Looks like you added this to the wrong handler.

Reply:
Whoops 😅 embarrassing! Sorry, I tried for a couple hours to get the tests running, and couldn't manage it. I think these changes should fix the problem though (and I made the testing a little better)